include/models/overwrite.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef _overwrite_hpp_
+#define _overwrite_hpp_
+
+#include <models/entity.hpp>
+
+namespace disccord
+{
+    namespace models
+    {
+        class overwrite : public model

```

Comment:
Yeah, in general I think that anything with a snowflake should be an entity, but in some cases it doesn't really make sense semantically.